easypost/shipment.py

```diff
@@ -19,11 +20,24 @@
 
 
 class Shipment(AllResource, CreateResource):
-    def regenerate_rates(self) -> "Shipment":
+    @classmethod
+    def create(cls, api_key: Optional[str] = None, carbon_offset: Optional[bool] = False, **params) -> "Shipment":
+        """Create an Shipment object."""
+        requestor = Requestor(local_api_key=api_key)
+        url = cls.class_url()
+        wrapped_params = {cls.snakecase_name(): params}
+        wrapped_params["carbon_offset"] = carbon_offset  # type: ignore
```

Comment:
This should fix the need to ignore the type while providing type safety.

```suggestion
        wrapped_params = {cls.snakecase_name(): params}  # type: Dict[str, Any]
        wrapped_params["carbon_offset"] = carbon_offset
```